docs/api/hooks/useSearchParams.md

```diff
@@ -52,46 +64,3 @@ useSearchParams([["tab", "1"]]);
 useSearchParams(new URLSearchParams("?tab=1"));
 ```
 
-## SetSearchParams Function
```

Comment:
"`SetSearchParams` function" & "Notes" aren't really additional examples, so we probably need to think about a generic way to still include these parts?

CC/ @brookslybrand @brophdawg11